static/index.xhtml

```diff
@@ -970,6 +982,15 @@
 			}
 		}
 
+		function stopTask() {
+			let taskId = document.getElementById("task_id").value;
+			fetch(`/task/stop?id=${taskId}`).then(response => {
+				if (response &amp;&amp; response.status == 200) {
+					alert("Task Stopped!!!")
```

Comment:
Can we remove this alert ... the status will update automatically when the task update is sent from the server.

Reply:
Fixed